### PR TITLE
bpo-45516: fix Traversable.name documentation

### PR DESCRIPTION
**Pull request by FFY00**
Here we go again... Sorrry!

Signed-off-by: Filipe Laíns <lains@riseup.net>

https://bugs.python.org/issue45516

Automerge-Triggered-By: GH:jaraco

### CODE REVIEW

File: Doc/library/importlib.rst [hunk 1]
```diff
@@ -815,7 +815,7 @@ ABC hierarchy::
 
     .. versionadded:: 3.9
 
-    .. abstractmethod:: name()
+    .. property abstractmethod:: name()
```
Comment (jaraco) [inline Doc/library/importlib.rst]: Is `property abstractmethod` the right syntax?  I don't see that anywhere else in the docs.

Comment (jaraco) [reply Doc/library/importlib.rst]: Indeed, this syntax seems not to be supported as the block doesn't render anything:

image

Comment (jaraco) [reply Doc/library/importlib.rst]: I found [one other example](https://github.com/python/cpython/blame/086b5c6ce16b524629428b4fd5c9213930bfca47/Doc/library/numbers.rst#L33-L35) of an abstract property. It's documented as an attribute with "Abstract" in the text. Let's follow that pattern until there's something better.

Comment (jaraco) [reply Doc/library/importlib.rst]: Fixed in f89e2c037a.

**Review by jaraco — APPROVED** [review]

**Comment by ewjoachim** [comment]
Ha, found the same bug on my side and I was on my way to fix it, but it's already fixed. Just wanted to thank you all folks ! :heart:

**Pull request merged**